src/apps.rs

```diff
@@ -22,16 +22,19 @@ pub fn create(config: &Config, app: &AppId, data: serde_json::Value) -> Result<(
         }
     });
 
-    let res = client
+    match client
         .post(&url)
         .header(reqwest::header::CONTENT_TYPE, "application/json")
         .body(body.to_string())
         .bearer_auth(&config.token.access_token().secret())
         .send()
-        .context("Can't create app.")?;
-
-    util::print_result(res, format!("App {}", app), Verbs::create);
-    Ok(())
+        .context("Can't create app.") {
+            Ok(r) => util::print_result(r, format!("App {}", app), Verbs::create),
+            Err(e) => {
+                eprintln!("{}", e);
+                std::process::exit(3);
```

Comment:
I'd rather keep the `Result` as the return type for the function and here do something like : 
```
.map(|res| util::print_result(res, format!("App {}", app), Verbs::create))
```
That way we let reqwest generate the error and the appropriate status code